@types/jsb.d.ts

```diff
@@ -705,137 +749,797 @@ declare namespace jsb {
+         *
+         * @zh
+         * 当前设备上的 Play Store 不支持所请求的功能。
          */
-        getBillingConfigAsync(): void;
-
+        SERVICE_UNAVAILABLE = 2,
+        /**
+         * @en
+         * A user billing error occurred during processing.
+         *
+         * @zh
+         * 处理过程中出现用户billing错误。
+         */
+        BILLING_UNAVAILABLE = 3,
         /**
-         * @en Creates alternative billing only purchase details that can be used to report a transaction made via alternative billing without user choice to use Google Play billing.
-         * @zh 创建仅限替代结算的购买详情，可用于报告通过替代结算进行的交易，而无需用户选择使用 Google Play Billing。
+         * @en
+         * The requested product is not available for purchase.
+         *
+         * @zh
+         * 所请求的产品无法购买。
          */
-        createAlternativeBillingOnlyReportingDetailsAsync(): void;
+        ITEM_UNAVAILABLE = 4,
         /**
-         * @en Checks the availability of offering alternative billing without user choice to use Google Play Billing.
-         * @zh 检查是否可以提供替代结算方式，而无需用户选择使用 Google Play Billing方式。
+         * @en
+         * Error resulting from incorrect usage of the API.
+         *
+         * @zh
+         * 由于错误使用 API 而导致的错误。
          */
-        isAlternativeBillingOnlyAvailableAsync(): void;
+        DEVELOPER_ERROR = 5,
+        /**
+         * @en
+         * Fatal error during the API action.
+         *
+         * @zh
+         * API 操作期间发生致命错误。
+         */
+        ERROR = 6,
+        /**
+         * @en
+         * The purchase failed because the item is already owned.
+         *
+         * @zh
+         * 购买失败，因为该物品已被拥有。
+         */
+        ITEM_ALREADY_OWNED = 7,
+        /**
+         * @en
+         * Requested action on the item failed since it is not owned by the user.
+         *
+         * @zh
+         * 由于该项目不属于用户，因此对该项目请求的操作失败。
+         */
+        ITEM_NOT_OWNED = 8,
+        /**
+         * @en
+         * A network error occurred during the operation.
+         *
+         * @zh
+         * 操作期间发生网络错误。
+         */
+        NETWORK_ERROR = 12,
+    }
+    /**
+     * @en
+     * Features/capabilities supported by isFeatureSupported.
+     *
+     * @zh
+     * 支持的特性/能力isFeatureSupported。
+     */
+    export enum FeatureType {
+        /**
+         * @en
+         * Purchase/query for subscriptions.
+         *
+         * @zh
+         * 购买/查询订阅。
+         */
+        SUBSCRIPTIONS = 'subscriptions',
+        /**
+         * @en
+         * Subscriptions update/replace.
+         *
+         * @zh
+         * 订阅更新/替换。
+         */
+        SUBSCRIPTIONS_UPDATE = 'subscriptionsUpdate',
+        /**
+         * @en
+         * Launch a price change confirmation flow.
+         *
+         * @zh
+         * 启动价格变动确认流程。
+         */
+        PRICE_CHANGE_CONFIRMATION = 'priceChangeConfirmation',
+        /**
+        * @en
+        * Show in-app messages.
+        *
+        * @zh
+        * 显示应用内消息。
+        */
+        IN_APP_MESSAGING = 'bbb',
```

Comment:
What's `bbb`? typo? Should be `'inAppMessaging'`?
Also don't know what the following `fff`, `ggg`, `jjj` stand for.

Reply:
https://developer.android.com/reference/com/android/billingclient/api/BillingClient.FeatureType